src/main/drivers/barometer/barometer_bmp388.h

```diff
@@ -0,0 +1,130 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * BMP388 Driver author: Dominic Clifton
+ *
+ * References:
+ * BMP388 datasheet - https://ae-bst.resource.bosch.com/media/_tech/media/datasheets/BST-BMP388-DS001.pdf
+ * BMP3-Sensor-API - https://github.com/BoschSensortec/BMP3-Sensor-API
+ * BMP280 Cleanflight driver
+ */
+
+#pragma once
+
+#define BMP388_I2C_ADDR                                 (0x76) // same as BMP280/BMP180
```

Comment:
Are all of these used outside of `barometer_bmp388.c`? If not, the ones not used should not be made part of the API, and go into `barometer_bmp388.c`.

Reply:
they are not, but this follows the pattern of the BMP280 driver so there is consistency in the BMP baro drivers.

Reply:
In general, I think we should be trying to design new code as cleanly as possible. The fact that legacy code is using an antipattern should not be a reason to deviate from this.

Reply:
no worries, I'll fix the other baro drivers too.